micro-rdk/src/common/app_client.rs

```diff
@@ -296,39 +296,6 @@ impl AppClient {
             None
         };
 
-        #[cfg(feature = "esp32")]
-        {
-            // In many cases when the ESP32 reboots the RTC clock will not be cleared and will retain the previous
-            // time. If the reboot was caused by a WDT timeout there is a chance the clock would drift significantly
-            // The new logic is to always set the clock once per boot
-
-            if let Some(current_dt) = datetime {
-                CLOCK_SET.call_once(|| {
```

Comment:
If this does end up being entirely removed, remember to remove the declaration of `CLOCK_SET` above.